cmd/kubeadm/app/util/config/initconfiguration.go

```diff
@@ -227,6 +228,19 @@ func BytesToInternalConfig(b []byte) (*kubeadmapi.InitConfiguration, error) {
 		if kubeadmutil.GroupVersionKindsHasInitConfiguration(gvk) {
 			// Set initcfg to an empty struct value the deserializer will populate
 			initcfg = &kubeadmapi.InitConfiguration{}
+
+			// Check for duplicate and unknown fields using yaml.UnmarshalStrict()
+			var initcfgTest interface{}
+			switch gvk.Version {
```

Comment:
this seems weird to dupe this in multiple locations when all serialized objects have Meta.